code/datums/materials/alloys.dm

```diff
@@ -0,0 +1,124 @@
+/** Materials made from other materials.
+  */
+/datum/material/alloy
+	name = "alloy"
+	desc = ""
+	/// The materials this alloy is made from weighted by their ratios.
+	var/list/composition = null
+	/// Breakdown flags required to reduce this alloy to its component materials.
+	var/req_breakdown_flags = BREAKDOWN_ALLOYS
+
+/datum/material/alloy/return_composition(amount=1, breakdown_flags)
+	if(req_breakdown_flags & !(breakdown_flags & req_breakdown_flags))
+		return ..()
+
+	. = list()
+	var/list/cached_comp = composition
+	for(var/comp_mat in cached_comp)
+		var/datum/material/component_material = SSmaterials.GetMaterialRef(comp_mat)
+		var/list/component_composition = component_material.return_composition(cached_comp[comp_mat], breakdown_flags)
+		for(var/comp_comp_mat in component_composition)
+			.[comp_comp_mat] += component_composition[comp_comp_mat] * amount
+
+
+/** Plasteel
+  *
+  * An alloy of iron and plasma.
+  */
+/datum/material/alloy/plasteel
+	name = "plasteel"
+	desc = "The heavy duty result of infusing iron with plasma."
+	color = "#828282"
+	value_per_unit = 0.135
+	strength_modifier = 1.5
+	integrity_modifier = 1.8 // Heavy duty.
+	armor_modifiers = list(MELEE = 1.5, BULLET = 1.5, LASER = 1.3, ENERGY = 1.3, BOMB = 2, BIO = 1, RAD = 1.8, FIRE = 1.5, ACID = 1.5)
```

Comment:
knight suit with 75 melee armor go brrrrrrrrr

Reply:
It's been nerfed... slightly. Plasteel also now add slowdown to any items it's included in. Is this sufficient?

Reply:
I just checked and it turns out plastic has a 50% boost to melee armor for some reason. It just feels wrong for plasteel to have less melee armor than plastic.

Reply:
the hell? I'd say nerf plastic,